package-lock.json

```diff
@@ -74099,6 +74154,26 @@
 				"is-windows": "^1.0.0"
 			}
 		},
+		"cross-fetch": {
+			"version": "4.0.0",
+			"resolved": "https://registry.npmjs.org/cross-fetch/-/cross-fetch-4.0.0.tgz",
+			"integrity": "sha512-e4a5N8lVvuLgAWgnCrLr2PP0YyDOTHa9H/Rj54dirp61qXnNq46m82bRhNqIA5VccJtWBvPTFRV3TtvHUKPB1g==",
+			"dev": true,
+			"requires": {
+				"node-fetch": "^2.6.12"
+			},
+			"dependencies": {
+				"node-fetch": {
+					"version": "2.7.0",
```

Comment:
`node-fetch` is also worth upgrading at the top level. Pinned to 2.6.7 there, causing duplication.

Reply:
Started work in https://github.com/WordPress/gutenberg/pull/65957 — will mark as ready for review in the next days.